lib/rules/keyframe-selector-notation/__tests__/index.mjs

```diff
@@ -193,6 +248,18 @@ testRule({
 			code: '@keyframes foo { 0%, 100% {} }',
 			description: 'selector lists',
 		},
+		{
+			code: stripIndent`
+				@keyframes foo {
+					/* stylelint-disable-next-line keyframe-selector-notation */
+					from, /* stylelint-disable-next-line keyframe-selector-notation */
+					from,
+					/* stylelint-disable-next-line keyframe-selector-notation */
+					from
+					{}
+				}
+			`,
+		},
 		...ACCEPT_TIMELINE_RANGE_NAME_TEST_CASES,
 	],
```

Comment:
question

Should it also handles typos when at least one keyword match?
Could you add a failing test for `@keyframes foo { to {}, fro {} }`?
i.e. I would expect a report for `to`

Reply:
Nothing changed with respect to this aspect.
Is there a specific bit of code that makes you suspect an issue?

Reply:
I felt that we were missing regression tests.
If you think it's not necessary, don't bother.